src/tests/unit/VerifyAccountHandler.test.ts

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable max-lines-per-function */
+/* eslint-disable @typescript-eslint/unbound-method */
+import { mock } from "jest-mock-extended";
+import { CONTEXT } from "./data/context";
+import { VALID_VERFIY_ACCOUNT } from "./data/verify-account-events";
+import { HttpCodesEnum } from "../../models/enums/HttpCodesEnum";
+import { MessageCodes } from "../../models/enums/MessageCodes";
+import { Constants } from "../../utils/Constants";
+import { VerifyAccountRequestProcessor } from "../../services/VerifyAccountRequestProcessor";
+import { lambdaHandler, logger } from "../../VerifyAccountHandler";
+
+const mockedVerifyAccountRequestProcessor = mock<VerifyAccountRequestProcessor>();
+
+describe("VerifyAccountHandler", () => {
+	let loggerSpy: jest.SpyInstance;
+  
+	beforeEach(() => {
+		loggerSpy = jest.spyOn(logger, "error");
```

Comment:
We have multiple ways we're mocking the logger in the BE might be worth aligning them at some point - not for this ticket